.circleci/config.yml

```diff
@@ -84,7 +84,7 @@ jobs:
           working_directory: tmconfig/
           command: |
             IMAGE_REPO=$(sed -n -e 's/^IMAGE_REPO[[:space:]].*=[[:space:]]*\(.*\)$/\1/p' ~/project/Makefile)
-            for cmd in $(sed -n -e 's/^COMMANDS[[:space:]]*=[[:space:]]*\(.*\)$/\1/p' ~/project/Makefile); do
+            for cmd in $(ls ~/project/cmd/); do
               sed -i overlays/<< parameters.cluster >>/triggermesh-core/deployments.yaml \
                 -e "s|gcr.io/triggermesh\(-private\)\?/${cmd}:.*|${IMAGE_REPO}/${cmd}:${CIRCLE_TAG:-${CIRCLE_SHA1}}|g"
             done
```

Comment:
Sorry can't comment on it, but there is also a "aws" at line 101 that needs to be removed.

Reply:
The issue with that syntax is it will include `~/project/cmd/` in the listing. If I have to adopt this syntax then the block will have to include substitution to trip that part